homeassistant/components/flume/entity.py

```diff
@@ -10,6 +12,13 @@
 from .const import DOMAIN
 
 
+@dataclass
+class FlumeNotificationSensorRequiredKeysMixin:
```

Comment:
We should keep platform specific objects in their respective files.

Otherwise I think you can rename this to be more generic

Reply:
This gets used by both binary and normal sensor so I wasn't sure where to put it. Suggestions?

Reply:
I'd rename it to `FlumeNotificationRequiredKeysMixin` so its not sensor specific

Reply:
Looks like this was marked as resolve but no code change.  Did you forget to push?

Reply:
I'm on my phone at the playground so yea prob an accident click. I'll get to it by Monday likely tomorrow

Reply:
MONDAY IT IS!!! 🎆 🎆 🎆 🎆